phpstan.neon.dist

```diff
@@ -79,7 +79,6 @@ parameters:
         -
             message: '~Parameter.*\$database of method.*expects string, string\|null given\.~'
             path: src/Schema/AbstractSchemaManager.php
-            count: 3
```

Comment:
💡 This error occurs only once currently. Since we don't match the exact count for other errors, I decided to remove the count here so we don't need to keep it up to date.